public/index.css

```diff
@@ -174,13 +174,18 @@ button > i {
 
 .project--technologies-bar {
     display: flex;
+    width: 70px;
     height: 8px;
     margin-top: 5px;
     border-radius: 5px;
     background-color: white;
```

Comment:
I think that this line is not needed. Once you remove it, you will not have the annoying small part on the right of the bar.

Reply:
if i remove it, then the length of the bars won't be the same.

Reply:
this comment reffers to " background-color: white;". Comment usually refers to te same line it was added